node/config/environ.sls

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/jinja/map.jinja" import node with context %}
+{%- from tplroot ~ "/jinja/libtofs.jinja" import files_switch with context %}
```

Comment:
This will break things from an organisational standpoint.  I've currently got the `ssf-formula` managing around 65 formulas across the org, to keep a standard structure throughout.  That includes managing `libtofs.jinja`.  Also note that were this change to be made, `docs/TOFS_pattern.rst` would be pointing to the wrong location.

Reply:
Of course, this idea could be discussed in the `wg` and adopted, if there is an appetite for that.

Reply:
np - I'll update two other formulas. Niels supported the `./yaml/` idea in the past but this is superficial stuff so I'll follow current style.